File: Python-Tasks/Mostafa-Hatem/Tasks_From_021_To_023/Task01.py



friends = ["Osama", "Ahmed", "Sayed", "Ali", "Mahmoud"]

print(f"Method One: {friends[0]}")
print(f"Method Two: {friends[friends.__len__() - friends.__len__()*2 ]}")
print(f"Method One: {friends[-5]}")
print(f"Method Two: {friends[0:1]}")
print(f"Method One: {friends[4]}")
print(f"Method Two: {friends[-1]}")

print(friends.__len__())

# Needed Output
# "Osama" => Method One
# "Osama" => Method Two
# "Mahmoud" => Method One
# "Mahmoud" => Method Two

File: Programing/C++_Tasks/Mostafa-Hatem/Tasks_From_47_To_54/Task01.cpp

#include <array>
#include <iostream>
using namespace std;

int main()
{
    cout << "// For Output\n";
    for (int i = 0; i < 11; i++)
    {
        cout << i << endl;
    }

    cout << "// While Output\n";
    int i = 0;
    while (i < 11)
    {
        cout << i << endl;
        i++;
    }

    cout << "// Do While Output\n";
    i = 0;
    do
    {
        cout << i << endl;
        i++;
    } while (i < 11);
    return 0;
}